CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing
+
+If you want to contribute to the project, you can find some helpful information here.
+
+## How can I help?
+
+Thanks for taking interest in helping us develop FS2!
+
+We welcome all kinds of contribution, including but not limited to:
+
+- 📖 documentation improvements, explanatory images/diagrams, fixes in typos, useful links
+- 🧹 refactorings of messy code, build structure, increasing test coverage or quality
+- 🚀 new features and bugfixes (including [bug reports and feature requests][fs2-issues]).
+
+Writing documentation is valuable for learning, so if you find some explanation insufficient, overly complicated or incorrect, it's a perfect opportunity to make a change to it!
+
+If at any point you run into problems, you can always ask a question on [the fs2-dev Gitter channel][fs2-dev].
+
+## How to submit a change
+
+If you see something worth adding, make the relevant changes in a fork of the source code and [submit a pull request to the project](fs2-pulls). If you don't know what you could help with, take a look at [the issues marked as "help wanted"][low-hanging-fruit] or ask on [Gitter][fs2-dev].
+
+We follow similar rules to [the cats-effect project's](https://github.com/typelevel/cats-effect#development).
+Most importantly, any contributions are expected to be made in the form of [GitHub pull requests to the FS2 repository][fs2-pulls].
+Usually it takes two approvals for a change to be merged. If it takes too long to get it approved, feel free to ask on [the Gitter channel][fs2-dev].
+
+Remember to follow the [code of conduct][coc] in online and offline discourse.
+
+## Building the project locally
+
+### Prerequisites
+
+You'll need JDK 11 and [sbt][sbt].
+
+We use several sbt plugins to build and check the project, including [MiMa (Migration Manager)][mima], [scalafmt][scalafmt] and [sbt-microsites][sbt-microsites].
+
+### Build process
+
+To compile the code for the whole repository, you can start an interactive sbt shell:
+
+```bash
+$ sbt
+[info] Loading global plugins from /Users/contributor/.sbt/1.0/plugins
+[info] Loading settings for project fs2-build from plugins.sbt,metals.sbt ...
+[info] Loading project definition from /Users/contributor/dev/fs2/project
+[info] Loading settings for project root from version.sbt,build.sbt ...
+[info] Set current project to root (in build file:/Users/contributor/dev/fs2/)
+sbt:root>
+```
+
+Inside the shell, you can compile the sources for the currently selected Scala version using the `compile` command.
+To compile the code for all Scala versions enabled in the build, use `+compile`. To include tests, `test:compile` or `+test:compile`, accordingly.
+
+
+### Testing
+
+To test the code, you can run the `testJVM` or `testJS` command, depending on the platform you want to run the tests on.
```

Comment:
are there commands to run _all_ tests, or do you _have_ to run `testJVM` _and_ `testJS`?

Reply:
You can just do `test`. I'll mention that...